qemu/tests/cgroup.py

```diff
@@ -169,27 +170,27 @@ def rm_scsi_disks(no_disks):
         :param no_disks: How many disks to remove
         :note: params['cgroup_rmmod_scsi_debug'] == "yes" => rmmod scsi_debug
         """
-        utils.system("echo -%d > /sys/bus/pseudo/drivers/scsi_debug/add_host"
-                     % no_disks)
+        process.system("echo -%d > /sys/bus/pseudo/drivers/scsi_debug/add_host"
+                       % no_disks)
 
         if params.get('cgroup_rmmod_scsi_debug', "no") == "yes":
-            utils.system("rmmod scsi_debug")
+            process.system("rmmod scsi_debug")
 
     def param_add_scsi_disks():
         """
         Adds scsi_debug disk to every VM in params['vms']
         :param prefix: adds prefix to drive name
         """
-        if utils.system("lsmod | grep scsi_debug", ignore_status=True):
-            utils.system("modprobe scsi_debug dev_size_mb=8 add_host=0")
+        if process.system("lsmod | grep scsi_debug", ignore_status=True):
+            process.system("modprobe scsi_debug dev_size_mb=8 add_host=0")
         for name in params['vms'].split(' '):
             disk_name = "scsi-debug-" + name
             utils.system("echo 1 >/sys/bus/pseudo/drivers/scsi_debug/add_host")
             time.sleep(1)   # Wait for device init
-            dev = utils.system_output("ls /dev/sd* | tail -n 1")
+            dev = process.system_output("ls /dev/sd* | tail -n 1")
```

Comment:
`dev = process.system_output("ls /dev/sd* | tail -n 1", shell=True)` else you might hit,
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/avocado_framework-46.0-py2.7.egg/avocado/utils/process.py", line 1211, in system_output
    sudo=sudo, ignore_bg_processes=ignore_bg_processes)
  File "/usr/lib/python2.7/site-packages/avocado_framework-46.0-py2.7.egg/avocado/utils/process.py", line 1111, in run
    raise CmdError(cmd, sp.result)
```